lib/rack/reducer/refinements.rb

```diff
@@ -35,6 +35,7 @@ def symbolize_keys
           end
         end
 
+        alias_method :deep_symbolize_keys, :symbolize_keys
```

Comment:
Adding an alias here works well, but I actually think `deep_symbolize_keys` is a good name for this method in all cases. What about just renaming `symbolize_keys` to `deep_symbolize_keys` above? This entire module is private, so I'm okay with changing its internals.

Reply:
Thanks for providing the specs. Yes, I think renaming the method makes more sense in this case. Just pushed it 👍